secure_mail/management/commands/email_signing_key.py

```diff
@@ -70,8 +70,11 @@ def handle(self, *labels, **options):
         if options.get('generate') and labels:
             raise CommandError("You cannot specify fingerprints and "
                                "--generate when running this command")
-
+        if options.get('generate') and not SIGNING_KEY_DATA["Passphrase"])
+            raise CommandError("SIGNING_KEY_DATA needs Passphrase")
```

Comment:
This exception needs to be worded better. Something like

> SIGNING_KEY_DATA is missing a value for 'passphrase'. You must specify a value for the 'passphrase' key in the SIGNING_KEY_DATA setting variable.

makes it clear to users what the problem is, and more importantly, what they can do to fix it.